src/ucp/proto/proto_common.h

```diff
@@ -114,6 +114,11 @@ ucp_proto_common_find_lanes(const ucp_proto_common_init_params_t *params,
                             ucp_lane_index_t *lanes, ucp_md_map_t *reg_md_map_p);
 
 
+ucp_lane_index_t
+ucp_proto_common_find_am_bcopy_lane(const ucp_proto_init_params_t *params,
+                                    ucp_lane_type_t lane_type);
```

Comment:
why do you need to provide `lane_type` here?

Reply:
can it be TAG/AM/RMA_SW/AMO_SW types?

Reply:
ah, I see
it's AM lane that can be used for emulation and AM